README.md

```diff
@@ -138,7 +159,9 @@ LookerEmbedSDK.init('looker.example.com',
 
 ### Node helper
 
-A signing helper method `createSignedUrl()` is provided in
+If you prefer, your backend service can [implement the signature function](https://github.com/looker/looker_embed_sso_examples) instead of calling the Looker API by using a [Looker Embed secret](https://docs.looker.com/r/sdk/sso-embed). Manually generating the signed URL avoids a call to the Looker API but is more error prone.
+
+One example of a helper method that generates a signed URL, `createSignedUrl()`, is provided in
```

Comment:
Are we already providing a link to the other language examples somewhere? If not, this seems like a good place.

https://github.com/looker/looker_embed_sso_examples